MekHQ/src/mekhq/campaign/parts/equipment/MissingLargeCraftAmmoBin.java

```diff
@@ -121,7 +122,8 @@ public boolean isAcceptableReplacement(Part part, boolean refit) {
         // than an LargeCraftAmmoBin. Subclasses should use a similar check, which
         // breaks Composability to a degree but in this case we've used
         // subclasses where they're not truly composable.
-        if ((part instanceof LargeCraftAmmoBin) && (part.getClass() == LargeCraftAmmoBin.class)) {
+        if ((part instanceof LargeCraftAmmoBin)
+                && Objects.equals(part.getClass(), LargeCraftAmmoBin.class)) {
```

Comment:
Why the instanceof usage here? Wouldn't that be covered in the Objects.equals?

Reply:
I originally only had the get-class check on the base classes and just had instanceof in the subclasses, however I must not have doubled back and removed that here.